src/components/CrptoDetails.test.js

```diff
@@ -4,7 +4,7 @@ import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
 import configureStore from 'redux-mock-store';
 import CryptoDetails from './CryptoDetails';
-import '@testing-library/jest-dom'
+import '@testing-library/jest-dom';
 
 const mockStore = configureStore([]);
```

Comment:
- [Optional] Dear @Bleu-Yves-Sopoude, I notice you have moved your test files to the `src/components/` location, this is not the standard way of organizing your tests. Usually, it is recommended to save your test in the `src/test/` folder.